app-typescript/components/DatePicker.tsx

```diff
@@ -33,6 +33,7 @@ interface IDatePickerBase extends IInputWrapper {
         }}
      */
     locale?: DatePickerLocaleSettings;
+    hideClear?: boolean;
```

Comment:
Can you improve naming? [verb][verb] is a bit confusing. I'd use either `hideClearButton` or `disableClearing`